templates/items/details/details-class.hbs

```diff
@@ -15,18 +15,20 @@
 
     {{!-- Hit Dice --}}
     <div class="form-group split-group">
-        <label>{{ localize "DND5E.CLASS.FIELDS.hitDice.label" }}</label>
+        <label>{{ localize "DND5E.CLASS.FIELDS.hd.label" }}</label>
         <div class="form-fields">
 
             {{!-- Denomination --}}
-            {{ formField fields.hitDice value=source.hitDice options=hitDieTypes label="DND5E.Denomination"
+            {{ formField fields.hd.fields.denomination value=source.hd.denomination options=hitDieTypes label="DND5E.Denomination"
                          localize=true classes="label-top" }}
 
             {{!-- Spent --}}
-            {{ formField fields.hitDiceUsed value=source.hitDiceUsed placeholder="0" localize=true label="DND5E.Spent"
+            {{ formField fields.hd.fields.spent value=source.hd.spent placeholder="0" localize=true label="DND5E.Spent"
                          classes="label-top" }}
         </div>
     </div>
+
+    {{ formGroup fields.hd.fields.additional value=source.hd.additional localize=true }}
```

Comment:
I thought maybe this should be an AE-only field. There's really no precedence in the rules for classes having 'additional' hit dice that I'm aware of.

Reply:
I didn't look into whether that would result in complications during data prep, but I imagine it would.

Reply:
can probably get to it within this or next week but likely you can get to it before then so fine by me

Reply:
> I didn't look into whether that would result in complications during data prep, but I imagine it would.

An enchantment might be able to do it, but otherwise it would require redoing parts of the `HitDie` class - and an effect targeting `classes` would throw an error. I'm open to suggestions.

Reply:
Right, I keep forgetting that this is on an Item. Ignore my AE suggestion then. I meant more that it's fine to stay in the data model if something needs to adjust class hit dice, but I was less sure about it being available directly in the UI by default, since the game rules do not make use of it at all. Does that sound reasonable, or do you think it really needs to be in the UI by default?

Reply:
We've so far only excluded properties from the UI if they are not yet functional, isn't that case?

I don't feel so strongly about it if having a cleaner UI with fewer options is important, those who know about it can update a class item via API if need be, then.

Reply:
Though in that case, I think we should include it in the UI if it has a truthy value.